xml/upgrade-to-pacific.xml

```diff
@@ -0,0 +1,222 @@
+    &productnumber; software repositories, as well as the registry of container
+    images.
+   </para>
+   <sect3 xml:id="verify-previous-upgrade-patch-repos-repos-7p">
+    <title>Software repositories</title>
+    <para>
+     If all nodes are registered with SCC, you will be able to use the
+     <command>zypper migration</command> command to upgrade. Refer to
+     <link
+      xlink:href="https://documentation.suse.com/sles/15-SP3/html/SLES-all/cha-upgrade-online.html#sec-upgrade-online-zypper"/>
+     for more details.
+    </para>
+    <para>
+     If nodes are <emphasis role="bold">not</emphasis> registered with SCC,
+     disable all existing software repositories and add both the
+     <literal>Pool</literal> and <literal>Updates</literal> repositories for
+     each of the following extensions:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       SLE-Product-SLES/15-SP3
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       SLE-Module-Basesystem/15-SP3
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       SLE-Module-Server-Applications/15-SP3
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       SUSE-Enterprise-Storage-7.1
+      </para>
+     </listitem>
+    </itemizedlist>
+   </sect3>
+   <sect3 xml:id="verify-previous-upgrade-patch-repos-images-7p">
+    <title>Container images</title>
+    <para>
+     All cluster nodes need access to the container image registry. In most
+     cases, you will use the public SUSE registry at
+     <literal>registry.suse.com</literal>. You need the following images:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       registry.suse.com/ses/7.1/ceph/ceph
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       registry.suse.com/ses/7.1/ceph/grafana
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       registry.suse.com/ses/7.1/prometheus/prometheus-server
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       registry.suse.com/ses/7.1/prometheus/prometheus-node-exporter
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       registry.suse.com/ses/7.1/prometheus/prometheus-alertmanager
+      </para>
+     </listitem>
+    </itemizedlist>
+    <para>
+     Alternatively&mdash;for example, for air-gapped
+     deployments&mdash;configure a local registry and verify that you have the
+     correct set of container images available. Refer to
+     <xref
+     linkend="deploy-cephadm-configure-registry"/> for more details
+     about configuring a local container image registry.
+    </para>
+   </sect3>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="upgrade-cluster-nodes">
+  <title>Migrate &sls; on each cluster node to version &cephos;</title>
+
+  <para>
+   EMPTY
+  </para>
+ </sect1>
+ <sect1 xml:id="upgrade-ses-packages">
+  <title>Upgrade &productname; related packages on each cluster node</title>
+
+  <para>
+   EMPTY
```

Comment:
content?

Reply:
@mgfritch Yes, the question is how to upgrade them. Whether via `zypper dup` or `ceph-salt update`?

Reply:
let's document `ceph-salt update`, but that does have the precondition that the SES7.1 repo is setup correctly in zypper

Reply:
@mgfritch ses7.1 should be setup either automatically with SCC or manually, see https://github.com/SUSE/doc-ses/pull/987/files#diff-bfcae6e91cccf9ecc83a273dbb63c7d0cdb0b367b560722d4b2803d85ef68df5R148